Cargo.toml

```diff
@@ -12,6 +12,7 @@ serde = "1.0"
 toml = "0.4"
 directories = "0.10.0"
 serde_derive = { version = "1.0", optional = true }
+failure = "*"
```

Comment:
use the version instead of *

Reply:
Should I use something `failure = "0.1.5"` or `failure = "^0.1"` to let any upstream fixes of possible bugs filter through?

Reply:
`failure = "0.1"` should suffice since it is implicit `^0.1`

Reply:
> `failure = "0.1"` should suffice since it is implicit `^0.1`

I didn't know that. Thank you!